smartsim/_core/launcher/dragon/dragonLauncher.py

```diff
@@ -91,210 +71,47 @@ class DragonLauncher(WLMLauncher):
 
     def __init__(self) -> None:
         super().__init__()
-        self._context = zmq.Context()
-        self._timeout = CONFIG.dragon_server_timeout
-        self._reconnect_timeout = CONFIG.dragon_server_reconnect_timeout
-        self._startup_timeout = CONFIG.dragon_server_startup_timeout
-        self._dragon_head_socket: t.Optional[zmq.Socket[t.Any]] = None
-        self._dragon_head_process: t.Optional[subprocess.Popen[bytes]] = None
-        # Returned by dragon head, useful if shutdown is to be requested
-        # but process was started by another launcher
-        self._dragon_head_pid: t.Optional[int] = None
-        self._authenticator: t.Optional[zmq.auth.thread.ThreadAuthenticator] = None
-
-        self._set_timeout(self._timeout)
+        self._connector = DragonConnector()
+        self._slurm_launcher = SlurmLauncher()
+        self._pbs_launcher = PBSLauncher()
```

Comment:
It's only for Batch jobs and for the very unlikely situation where a user has both Slurm and PBS on the system and wants to launch batch jobs on both because he wants to test us.